bin/src/data/transform/data_transformation_generators.py

```diff
@@ -126,6 +126,52 @@ def transform_all(self, data: list) -> list:
             function_specific_input = [(item) for item in data]
             return pool.map(self.transform, function_specific_input)
         
+class GaussianChunk(AbstractAugmentationGenerator):
+    """
+    This augmentation strategy chunks the input sequences from the middle position +/- a value obtained through a gaussian distribution.
```

Comment:
```suggestion
    This augmentation strategy chunks the input sequences, for which the middle position is obtained through a gaussian distribution.
    
    In concrete, it changes the the middle position (ie. peak summit) to another position. This position is chosen based on a gaussian distribution, so the region close to the middle point are more likely to be chosen than the rest.
    Then a chunk with size `chunk_size` around the new middle point is returned for each sequence.
```
just make the comment more informative